app/main.py

```diff
@@ -1,8 +1,42 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, 
+    comfort_class: int, clean_mark: int, brand: str) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(self,
+                distance_from_city_center: float, 
+                clean_power: int,
+                average_rating: float,
+                count_of_ratings: int) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+        def serve_cars(cars: list) -> int:
+            total_income = 0
+            for car in cars:
+                if car.clean_mark < self.clean_power:
+                    income = car.comfort_class * 1.25
+                    total_income += income
+                    car.clean_mark = self.clean_power
+            return round(total_income, 1)
+
+        def calculate_washing_price(self, car: Car) -> float:
+            total_price = car.comfort_class * (self.clean_power - car.clean_mark) * self.average_rating / self.distance_from_city_center
+            return round(total_price, 1)
+
+        def wash_single_car(car: Car) -> None:
+            if car.clean_mark < self.clean_power:
+                car.clean_mark = self.clean_power
+        
+        def rate_service(rating: self.count_of_ratings) -> None:
+            if not (1 <= rating <= 5):
+                print("Rating must be between 1 and 5.")
+
+            count_of_ratings += 1
```

Comment:
The variable `count_of_ratings` should be prefixed with `self.` to correctly reference the instance variable. Change `count_of_ratings += 1` to `self.count_of_ratings += 1`.